cookbook/bundles/best_practices.rst

```diff
@@ -221,7 +218,7 @@ following standardized instructions in your ``README.md`` file.
     following command to download the latest stable version of this bundle:
 
     ```bash
-    $ composer require <package-name> "~1"
+    $ composer require <package-name>
```

Comment:
Is typing 2 more characters already seen as a decrease in DX? I think adding a 5 line long paragraph about a simple version number is not needed and makes the docs harder to read.

And indeed, we shouldn't promote it to use everywhere, but it should be used in the installation instructions. A case where a version isn't needed is: "This bundle can also work with xxx/yyy. You can install it using `composer require xxx/yyy`". However, this section isn't about that case.